terraform/application/config/staging.tfvars.json

```diff
@@ -1,4 +1,13 @@
 {
     "cluster": "test",
-    "namespace": "bat-staging"
+    "namespace": "bat-staging",
+    "statuscake_password_name": "SC-PASSWORD",
+    "infra_key_vault_name": "s189t01-ittms-stg-inf-kv",
+    "key_vault_resource_group":  "s189t01-ittms-stg-rg",
+    "statuscake_alerts": {
+      "alert": {
+        "website_url": [ "https://staging.claim-funding-for-mentor-training.education.gov.uk/healthcheck.json" ],
```

Comment:
Can we monitor the ingress URL as well?
Such as: https://github.com/DFE-Digital/teacher-services-cloud/blob/main/templates/new_service/terraform/application/statuscake.tf#L6

Reply:
done